### ISSUE TITLE
4-pin CCT LED strip (C-/C+/W-/W+)

### ISSUE THREAD

**Issue by tokamac** [xoste49/MJTD01SYL#2]
This is not an issue per se but there is no discussion section. 
The 1S version of the Xiaomi desk lamp provides 4 pins (C-/C+/W-/W+) to the LED strip, instead of 3 usually for CCT LED strips (WW, CW, common +) which was the case for the prior version. 
Using a multimeter, one can measure a voltage of 12VDC when the lamp is powered off, and each pair C-/C+ and W-/W+ goes up to 24VDC independently when the respective channel is put to maximum brightness.
Besides the electronics, what is that kind of CCT LED strip? Does it have a specific name and where can we find it?

**Comment by xoste49** [comment]
@tokamac maybe it 
![2022-07-11_104346](https://user-images.githubusercontent.com/7299412/178213954-a8f12e00-aa5c-48cf-8e82-baa97d838535.png)

**Comment by tokamac** [comment]
@xoste49  I don't think so, as although this LED strip is indeed a CCT w/ 4 wires, it clearly states on its product page that one needs to choose the voltage version: either DC12V, or DC24V, meaning it is a constant voltage LED strip, like 99% of them. Whereas each WW vs CW channel on the Mi desk lamp 1S independently varies from 12VDC (0% brightness for that channel) to 24VDC (100% brightness for that channel).

It's almost as if Xiaomi had not used a conventional CCT LED strip, but more like a dual system instead: two 2-wire constant-current LED strips (one 2-wire WW LED strip, and one 2-wire CW LED strip) combined into a single LED bar strip in the end, each with its own 2-channel control, aggregated through the firmware. Just a hypothesis of mine for now, I don't know if I am right.

But as the power adapter output is 12VDC constant voltage at max. 0.8A, there may be some conversion done by the board in the lamp.
